packages/test/stochastic-test-utils/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@fluidframework/stochastic-test-utils",
```

Comment:
does this need to be @fluidframework? this will lead to npm publishing? can we it be fluid-internal

Reply:
saw you comment above. i prefer to limit scope/surface area. it we have a need we can publish it later

Reply:
yeah, sounds reasonable to me.